tests/GeneralStateTestsRunner.js

```diff
@@ -42,6 +44,31 @@ function runTestCase (testData, t, cb) {
       }
 
       if (tx.validate()) {
+        if (options.JSONTrace) {
+          vm.on('step', function (e) {
+            hexStack = []
+            for (var i = 0; i < e.stack.length; i++) {
+              var hexVal = ethUtil.stripZeros(e.stack[i]).toString('hex')
```

Comment:
Instead of this code you could use `'0x' + new BN(e.stack[i]).toString(16, 0)` (second parameter is padding, set to 0 to mimic the bad requirement of 4-bit alignment in Ethereum, e.g. using 0x0 and not 0x00)